src/core/simulator/kitchen_simulator.py

```diff
@@ -0,0 +1,39 @@
+# Kitchen Simulator impl
+"""
+    This file will be use to implement a kitchen simulator to work will the different orders, that will be manages
+    for the differences chef in kitchen, they will manage the orders status while they are working with them. Also
+    the implementation of threads will be used to work at the same time with the kitchen an new orders in real time.
+"""
+from time import sleep
+from src.core.ioc import get_ioc_instance
+from threading import Thread
+
+
+class KitchenSimulatorSetup:
+    def __init__(self):
+        self.instance_ioc_kitchen = get_ioc_instance()
+
+
+class KitchenSimulator:
+    def __init__(self):
+        self.kitchen_setup = KitchenSimulatorSetup()
+
+    def kitchen_process(self):
```

Comment:
Can make it more abstract? I don't want the kitchen similator to know anything about the thread, maybe some external state variables or elapased time since the last call